### ISSUE TITLE
Multiple node labels are broken

### ISSUE THREAD

**Issue by Karry** [Framstag/libosmscout#195]
standard.oss stylesheet define two labels for natural_peak (name and elevation), but it don't work. When only one label is defined, it is displayed correctly, but when both are defined, nothing is shown.

It can be reproduced with map of Czech republic and "Železný vrch" peak http://www.openstreetmap.org/node/366820981

**Comment by Karry** [comment]
Nodes are dropped, because space between them is less than `labelSpace` (8px for me) and `LabelLayouter::Intersects` return true for them

```
--- Placing: 'Železný vrch' 150.76 - 247.385, 374.888 - 391.888
INSERT
--- Placing: '1321m' 172.705 - 225.439, 392.888 - 409.888
Event: 'Železný vrch' 150.76,374.888 | 150.76 - 247.385, 374.888 - 391.888
Event: 'Železný vrch' 150.76,391.888 | 150.76 - 247.385, 374.888 - 391.888
'Železný vrch' 150.76,374.888 | 150.76 - 247.385, 374.888 - 391.888
DROPPING same prio and exit 'Železný vrch' 29 vs. 29
```

**Comment by Karry** [comment]
Hmm. I see two possibilities: 

- push label origin object from draw methods to `MapPainter::LayoutPointLabels` -> `LabelLayouter::Placelabel` and ignore intersections for labels from same origin.
- extend `LabelData` to hold multi-line labels

I like second option more, it may be used for multiline area labels later... First solution is nasty.

**Comment by Framstag** [comment]
I have now pushed a fix for this. Case was that the GetTextDimension() method returned  bigger height, that GetFontSize(). So layout broke, because the label regions overlapped in result. Also the use of GetFontSize() would break things anyway, if we support multiline labels.

I now introduced a id to the LabelData. LabelData instances with the some id do not intersect per definition. So the layouters collision detection is disabled for multiple labels from the some object.
